### ISSUE TITLE
Screen Reader not announcing Bryntum gantt cell data in Windows

### ISSUE THREAD

**Issue by taauntik** [bryntum/support#11769]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=51&t=33797&p=172377#p172377)

Hi there,

In Bryntum Gantt, the cell information is not accessible to the users. When we are moving the focus to any data cell, the Screen Reader is only announcing the column header name and not the current cell value. The screen reader should announce both the column header name and its value. 

Testing environment - Windows + Chrome + NVDA.

![bryntum_gantt_cell_announcement_issue.png](https://forum.bryntum.com/download/file.php?id=35979)
